test_server.py

```diff
@@ -1,12 +1,18 @@
 import enet
-
 SHUTDOWN_MSG = "SHUTDOWN"
 
 host = enet.Host(enet.Address(b"localhost", 54301), 10, 0, 0, 0)
+def receive_callback(address, data):
+    global host
+    if data and data == "\xff\xff\xff\xffgetstatus\x00":
```

Comment:
Have you tested this with both python2 and 3? Do you need to use a bytestring literal here?